pkg/tests/ossm/operator/clusterwide_mode_test.go

```diff
@@ -305,6 +305,78 @@ spec:
 			})
 		})
 
+		t.NewSubTest("verify that namespaces without istio-enable label are not included to the SMMR list").Run(func(t test.TestHelper) {
+			t.LogStep("Create 5 member namespaces")
+			createMemberNamespaces(t, 5)
+
+			t.LogStep("Wait for SMMR to be Ready")
+			oc.WaitSMMRReady(t, meshNamespace)
+			t.Logf("Actual SMMR members: %s", meshNamespace)
+
+			t.LogStep("Remove istio-injection=enabled label from member-1 and member-3 namespaces")
+			for _, member := range []string{"member-0", "member-2", "member-4"} {
+				t.Logf("Removing label from namespace %s", member)
+				oc.RemoveLabel(t, "", "Namespace", member, "istio-injection")
+			}
+
+			t.LogStep("Add other variation of istio-injection value to member-2 and member-4 namespace")
+			t.Log("Add istio-injection=disabled label to member-2 namespace")
+			oc.Label(t, "", "Namespace", "member-2", "istio-injection=disabled")
+			t.Log("Add istio-injection=notanoption label to member-4 namespace")
+			oc.Label(t, "", "Namespace", "member-4", "istio-injection=notanoption")
+
+			t.LogStep("Wait for SMMR to be Ready")
+			oc.WaitSMMRReady(t, meshNamespace)
+
+			t.LogStep("Check whether the SMMR shows the 2 namespaces created as members")
+			membersList := []string{"member-1", "member-3"}
+			notInMembersList := []string{"member-0", "member-2", "member-4"}
+			assertMembers(t, meshNamespace, membersList)
+			assertNonMembers(t, meshNamespace, notInMembersList)
+		})
+
+		t.NewSubTest("verify strict mTLS across service mesh members and not members").Run(func(t test.TestHelper) {
+			t.Log("Test strict mTLS across service mesh members")
+			t.Log("Doc: https://docs.openshift.com/container-platform/4.14/service_mesh/v2x/ossm-security.html#ossm-security-enabling-strict-mtls_ossm-security")
+
+			t.LogStep("Apply SMMR to select foo and bar as members")
+			oc.ApplyString(t, meshNamespace, fooBarSMMR)
+
+			t.LogStep("Install httpbin and sleep in multiple namespaces")
+			app.InstallAndWaitReady(t,
```

Comment:
Because we are adding this apps with the configuration we need to add a proper cleanup to delete everything set on this test to avoid failures in the next test case